src/main/java/org/example/springboot/model/Book.java

```diff
@@ -0,0 +1,27 @@
+package org.example.springboot.model;
+
+import jakarta.persistence.Column;
+import jakarta.persistence.Entity;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.GenerationType;
+import jakarta.persistence.Id;
+import java.math.BigDecimal;
+import lombok.Data;
+
+@Entity
+@Data
```

Comment:
Don't use `@Data` for entities, find an article about issues that could be appeared
```suggestion
@Getter
@Setter
@Table(name....)
```